servers/physics/body_sw.cpp

```diff
@@ -45,8 +45,9 @@ void BodySW::_update_transform_dependant() {
 	// update inertia tensor
 	Basis tb = principal_inertia_axes;
 	Basis tbt = tb.transposed();
-	tb.scale(_inv_inertia);
```

Comment:
Conventions have changed for the scale method, this was making the physics engine unstable